<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>button1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>button2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>button3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>button4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendClick">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>SendClick</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFileNameInit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>addFileNameInit</string>
    </property>
   </widget>
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fileDialog">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>fileDialog</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>137</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_y"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>138</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_x"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>SendClick</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button1_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button2_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button3_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button4_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addFileNameInit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addFileNameInit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>itemChicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileDialog</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileDialog_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendClick()</slot>
  <slot>button1_on()</slot>
  <slot>button2_on()</slot>
  <slot>button3_on()</slot>
  <slot>button4_on()</slot>
  <slot>addFileNameInit()</slot>
  <slot>itemChicked(QListWidgetItem*)</slot>
  <slot>fileDialog_on()</slot>
 </slots>
</ui>
